webpack.config.js

```diff
@@ -111,7 +111,7 @@ module.exports = (env) => ({
         'api/plugins/compiler': '/api/compiler-hooks/',
         'api/plugins/compilation': '/api/compilation/',
         'api/plugins/module-factories': '/api/module-methods/',
-        'api/plugins/parser': '/api/parser/',
+        'api/plugins/parser': '/api/parser-hooks/',
```

Comment:
You would also need to add another redirect from `/api/parser/` to the new url... I suggest you revert the file rename bcoz of SEO and extra redirect

Reply:
It would also allow to review the PR better. When its moved its a pain to see the diff vs new additions

Reply:
You are right, I've reverted the file renaming